service/src/main/java/bio/terra/externalcreds/services/PassportService.java

```diff
@@ -91,34 +97,62 @@ public ValidatePassportResultInternal validatePassport(
           if (visaComparator.visaTypeSupported(visa)
               && visa.getIssuer().equals(criterion.getIssuer())
               && visaComparator.matchesCriterion(visa, criterion)) {
-            var linkedAccount =
-                linkedAccountsByJwtId.get(passportWithVisas.getPassport().getJwtId());
             var auditInfoMap =
-                new HashMap<>(
-                    Map.of(
-                        "passport_jti",
-                        passportWithVisas.getPassport().getJwtId(),
-                        "external_user_id",
-                        linkedAccount.getExternalUserId(),
-                        "internal_user_id",
-                        linkedAccount.getUserId()));
+                new HashMap<>(Map.of("passport_jti", passportWithVisas.getPassport().getJwtId()));
```

Comment:
Is there a reason that you are wrapping the map in a hashmap?  Doesn't look like you ever edit it

Reply:
JK, just saw that you write to it.  Bummer that there's not a way to have `map.of` create a mutable map

Reply:
A `MutableMap.of()` feature would be lovely...

Reply:
lol, even better :)